server/images.go

```diff
@@ -40,18 +40,17 @@ type RegistryOptions struct {
 }
 
 type Model struct {
-	Name          string `json:"name"`
-	ShortName     string
-	ModelPath     string
-	OriginalModel string
-	AdapterPaths  []string
-	Template      string
-	System        string
-	License       []string
-	Digest        string
-	ConfigDigest  string
-	Options       map[string]interface{}
-	Embeddings    []vector.Embedding
+	Name          string                 `json:"-"`
+	ShortName     string                 `json:"-"`
+	ModelPath     string                 `json:"-"`
+	OriginalModel string                 `json:"originalModel"`
+	AdapterPaths  []string               `json:"adapterPaths"`
+	Template      string                 `json:"-"`
+	System        string                 `json:"system"`
+	License       []string               `json:"license"`
```

Comment:
Should these trigger a reload? I don't think so

Reply:
Removed license from the runnerDigest. system has to stay for now since we use it to calculate `num_keep`. 
```
if opts.NumKeep < 0 {
			promptWithSystem, err := model.Prompt(api.GenerateRequest{}, "")
			if err != nil {
				return err
			}

			promptNoSystem, err := model.Prompt(api.GenerateRequest{Context: []int{0}}, "")
			if err != nil {
				return err
			}

			tokensWithSystem, err := llmModel.Encode(ctx, promptWithSystem)
			if err != nil {
				return err
			}

			tokensNoSystem, err := llmModel.Encode(ctx, promptNoSystem)
			if err != nil {
				return err
			}

			opts.NumKeep = len(tokensWithSystem) - len(tokensNoSystem)

			llmModel.SetOptions(opts)
		}
```